Platform/AlderlakeBoardPkg/BoardConfigAdlS.py

```diff
@@ -329,18 +352,18 @@ def GetContainerList (self):
 
         if self.ENABLE_TCC:
             container_list.append (
-              ('TCCC', 'TccCacheCfg.bin' if os.path.exists(os.path.join(bins,   'TccCacheCfg.bin')) else '',   'Lz4', container_list_auth_type, 'KEY_ID_CONTAINER_COMP'+'_'+self._RSA_SIGN_TYPE, 0, self.TCC_CCFG_SIZE, 0),   # TCC Cache Config
+              ('TCCC', os.path.join(bins,  'TccCacheCfg.bin') if os.path.exists(os.path.join(bins,   'TccCacheCfg.bin')) else '',   'Lz4', container_list_auth_type, 'KEY_ID_CONTAINER_COMP'+'_'+self._RSA_SIGN_TYPE, 0, self.TCC_CCFG_SIZE, 0),   # TCC Cache Config
```

Comment:
Better split this line into 2 since it is too long, e.g. 
            TccCacheCfg  = os.path.join(bins,  'TccCacheCfg.bin') if os.path.exists(os.path.join(bins,  'TccCacheCfg.bin'))  else ''
            TccCrlBinary = os.path.join(bins,  'TccCrlBinary.bin')if os.path.exists(os.path.join(bins,  'TccCrlBinary.bin')) else ''
            TccStreamCfg = os.path.join(bins,  'TccStreamCfg.bin')if os.path.exists(os.path.join(bins,  'TccStreamCfg.bin')) else ''
            container_list.append (
              ('TCCC', TccCacheCfg,   'Lz4', container_list_auth_type, 'KEY_ID_CONTAINER_COMP'+'_'+self._RSA_SIGN_TYPE, 0, self.TCC_CCFG_SIZE, 0),  # TCC Cache Config
            )